package.json

```diff
@@ -21,6 +21,11 @@
   "engines": {
     "node": ">=12.2.0"
   },
+  "main": "./src/loader.js",
+  "bin": {
+    "presm": "./src/loader.js",
+    "presmbuilder": "./src/index.js"
```

Comment:
I think I would expect this to be `presm-build` (imperative) because it's a command.